### PR TITLE
NMA-491 (2) | Alert User (UI) | Hotfix

### PR DESCRIPTION
**Pull request by sambarboza**
## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Review by tomasz-ludek — APPROVED** [review]
Looks good!

**Comment by tomasz-ludek** [comment]
@sambarboza I've just noticed that it was already partially fixed by myself in https://github.com/dashevo/dash-wallet/pull/621 
I've just added the missing changes https://github.com/dashevo/dash-wallet/pull/621/commits/6580dc056c61f5e1006dc8c35cc2498df4952d77 so that we can close this ticket as replaced by #621

File: wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt [hunk 1]
```diff
@@ -139,10 +139,6 @@ class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root), Tex
         }
         searchContacts()
 
-        invite_friend_hint.setOnClickListener {
-            InviteFriendActivity.startOrError(requireActivity())
-        }
-
```
Comment (HashEngineering) [inline wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt:145]: This removes the listener for the invite Button on the Contacts Screen (when there are no contacts or contact requests)

**Comment by HashEngineering** [comment]
As stated by @tomasz-ludek all of these fixes were in #621.  There is no need to merge this PR.

**Pull request closed without merging**